stories/layout.stories.js

```diff
@@ -119,3 +117,18 @@ storiesOf('Components (Unthemed)|Layout', module)
       <CheckButton>This one is off</CheckButton>
     </>
   ))
+  .addWithJSX('StepsAccordion', () => (
+    <StepsAccordion>
+      <div isRequired={true}>
```

Comment:
It's a little weird to pass props to a component that it wouldn't support and then remove them internally. It seems like it would be cleaner to have an `AccordionStep` component instead.